src/views/Map/MainContent.js

```diff
@@ -277,7 +283,7 @@ const MainContent = ({mapStyle, mode, routes, directions}) => {
           variant="outlined"
         />
         <Box mt={2}>
-          <Button variant="outlined" onClick={handleSaveName}>{t('done')}</Button>
+          <Button variant="outlined" style={buttonColors} onClick={handleSaveName}>{t('done')}</Button>
```

Comment:
Los componentes de material-ui tienen una prop específica que se llama sx con la que te permiten acceder a cosas como el tema, el "spacing" por defecto, el tamaño del dispositivo, etc... Cambia style por sx (para el caso de uso que nos ocupa da igual pero para que te acostumbres) y mirate esto: 
[https://mui.com/material-ui/customization/how-to-customize/#the-sx-prop](url) (Cómo usarla)
[https://mui.com/system/properties/](url) (Propiedades y cómo los valores se traducen en css)

También puedes ver un montón de ejemplos en casi todos los proyectos, por ejemplo en geocomponents, para ver distintos casos de uso.